### ISSUE TITLE
Can the atom element labels be the same size?  See the image

### ISSUE THREAD

**Issue by digitalbio** [ncbi/icn3d#61]
I used Label > per Atom Element - this is great except the label for the Iron is much bigger than everything else.  Can these be the same size?

<img width="331" alt="Screen Shot 2021-11-06 at 5 48 08 PM" src="https://user-images.githubusercontent.com/6663433/140628219-15ab6ebf-a3b4-462a-a419-c0d3ec4bf44c.png">

**Comment by jiywang3** [comment]
This was fixed. Thanks for your report.

**Comment by digitalbio** [comment]
This one is not fixed yet.

Here's the information from the command window.
> clear all
> pickatom 3957(chain: 2DN2_D, residue: HIS, number: 92, atom: CE1)
> pickatom 4525(chain: 2DN2_HEM4, residue: HEM, number: 1003, atom: CGA)
> style proteins ball and stick
> style chemicals ball and stick
> add element labels

And here's what I did.
1.  Opened 2DN2
2.  I selected the histidine that's bound to the heme group.  
3.  I selected the heme group.
4.  Style menu and chose Proteins > Ball and Stick.
5.  Style menu and chose Chemicals > Ball and Stick.
6.  Analysis > Label > per Atom element

I still get a large label on the iron (Fe) and I'm unable to make the label on the iron the same size as the other labels.
<img width="432" alt="Screen Shot 2021-11-08 at 12 30 31 PM" src="https://user-images.githubusercontent.com/6663433/140813494-1e8b41cb-b8c7-47ef-a924-4f6957f57543.png">
